pkg/activator/net/throttler.go

```diff
@@ -326,6 +326,7 @@ func (t *Throttler) run(updateCh <-chan revisionDestsUpdate) {
 	for update := range updateCh {
 		t.handleUpdate(update)
 	}
+	t.logger.Info("The Throttler has stopped!")
```

Comment:
```suggestion
	t.logger.Info("The Throttler has stopped")
```

Do we even need this log?

Reply:
we have same messages for startup, so I guess we can for stoppage.